lcci.sample.yml

```diff
@@ -0,0 +1,5 @@
+#main:
+#  github-api-token: <TOKEN>
+volumes:
+  #TODO: Make it confurable via API
+  tools: lcci-tools
```

Comment:
Ideally I should add a parameter to the tool for that

Reply:
It should also be an absolute path, right?

Reply:
No need. Such definition will create a new named docker volume or write to an existing one if it exists.